network/p2p/p2pnode/libp2pNode_test.go

```diff
@@ -305,106 +283,6 @@ func TestCreateStream_SinglePairwiseConnection(t *testing.T) {
 	close(streamChan)
 }
 
-// TestCreateStream_SinglePeerDial ensures that the unicast manager only attempts to dial a peer once, retries dialing a peer the expected max amount of times when an
-// error is encountered and retries creating the stream the expected max amount of times when unicast.ErrDialInProgress is encountered.
-func TestCreateStream_SinglePeerDial(t *testing.T) {
```

Comment:
This test had a broken logic and was revamped as `TestUnicastManager_Dial_In_Progress_Backoff` in `manager_test.go`.